addons/website_sale/views/templates.xml

```diff
@@ -969,30 +969,32 @@
           Your cart is empty!
         </div>
         <t t-if="website_sale_order and website_sale_order.website_order_line">
-            <t t-foreach="website_sale_order.website_order_line" t-as="line">
-                <div class="row mb8 cart_line">
-                    <div class="col-3 text-center">
-                        <span t-field="line.product_id.image_128" t-options="{'widget': 'image', 'qweb_img_responsive': False, 'class': 'rounded o_image_64_max mb-2'}" />
-                    </div>
-                    <div class="col-9">
-                        <div>
-                            <t t-call="website_sale.cart_line_product_link">
-                                <span class="h6" t-esc="line.name_short" />
-                            </t>
+            <div class="o_wsale_cart_popover_product_lines">
```

Comment:
Could not we put the max-height on the popover itself instead of a new wrapper ?

Reply:
If I do that the Total price section below will be in the scroll and not visible unless scrolled to the bottom